### DISCUSSION TITLE
Unable to Type Master Password when using Autofill

### DISCUSSION THREAD

**Discussion by JeffResc** [dani-garcia/vaultwarden#1476] [General]
### Subject of the issue
Unable to Type Master Password when using Autofill

### Deployment environment

* bitwarden_rs version: 1.19.0

* Install method: Docker image

* Clients used:  iOS

* Reverse proxy and version: Traefik 2.4.5

* MySQL/MariaDB or PostgreSQL version: N/A

* Other relevant details: N/A

### Issue
I have removed the bottom portion of this template because I am cross referencing an issue from another repository, but I wanted to bring the issue to here as well just in case. Please let me know if this is not enough information as I would be happy to provide more.

Please see: bitwarden/mobile#1295

**Comment by BlackDex** [comment]
Thx for the link, i will move this to discussions since it's not something related to bitwarden_rs.
